SimpleBrowser/Browser.cs

```diff
@@ -765,7 +766,12 @@ public void BasicAuthenticationLogin(string domain, string username, string pass
 
             _basicAuthenticationTokens[domain] = new BasicAuthenticationToken(domain, username, password);
         }
-
+        
+        public void SetCertificate(System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates)

```

Comment:
Add System.Security.Cryptography.X509Certificates to the usings at the top of the file. Remove the namespace from here.